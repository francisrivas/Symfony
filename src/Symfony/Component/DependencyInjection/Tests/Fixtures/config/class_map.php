<?php

/*
 * This file is part of the Symfony package.
 *
 * (c) Fabien Potencier <fabien@symfony.com>
 *
 * For the full copyright and license information, please view the LICENSE
 * file that was distributed with this source code.
 */

namespace Symfony\Component\DependencyInjection\Loader\Configurator;

return new class() {
    public function __invoke(ContainerConfigurator $c)
    {
        $c->services()
            ->set('class_map_consumer', \stdClass::class)
                ->public()
                ->args([class_map(
                    'Symfony\Component\DependencyInjection\Tests\Fixtures\ClassMap\Valid',
                    realpath(__DIR__.'/../ClassMap/Valid'),
                )])
        ;
    }
};
