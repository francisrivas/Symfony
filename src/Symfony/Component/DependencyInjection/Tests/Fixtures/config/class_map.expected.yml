
services:
    service_container:
        class: Symfony\Component\DependencyInjection\ContainerInterface
        public: true
        synthetic: true
    class_map_consumer:
        class: stdClass
        public: true
        arguments: [{ 0: Symfony\Component\DependencyInjection\Tests\Fixtures\ClassMap\Valid\Bar, 1: Symfony\Component\DependencyInjection\Tests\Fixtures\ClassMap\Valid\Baz, 2: Symfony\Component\DependencyInjection\Tests\Fixtures\ClassMap\Valid\Corge, foo-attribute: Symfony\Component\DependencyInjection\Tests\Fixtures\ClassMap\Valid\Foo, foo-enum-attribute: Symfony\Component\DependencyInjection\Tests\Fixtures\ClassMap\Valid\FooEnum, 3: Symfony\Component\DependencyInjection\Tests\Fixtures\ClassMap\Valid\Qux }]
